content/library/api/connections/connections.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Connections and databases
+slug: /library/api-reference/connections
+---
+
+# Connections and databases
+
+## Setup your connection
+
+<TileContainer>
+<RefCard href="/library/api-reference/connections/st.experimental_connection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/experimental-connection.png" />
+
+#### Create a connection
+
+Connect to a database and return a connection object.
+
+```python
+st.experimental_connection('pet_db', type='sql')
+```
+
+</RefCard>
+</TileContainer>
+
+## Built-in connections
+
+<TileContainer>
+
+<RefCard href="/library/api-reference/connections/st.connections.sqlconnection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/sqlalchemy.png" />
+
+#### SQL connection
+
+Connect to a SQLAlchemy Session and return a connection object.
+
+```python
+conn = st.connections.SQL('pet_db')
+```
+
+</RefCard>
+
+<RefCard href="/library/api-reference/connections/st.connections.snowparkconnection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/snowflake.png" />
+
+#### Snowpark connection
+
+Connect to a Snowpark Session and return a connection object.
```

Comment:
#### SnowparkConnection

A connection to Snowflake Snowpark.

Reply:
Fixed on 16e925e